styles.css

```diff
@@ -0,0 +1,584 @@
+      }
+
+      .job-description {
+        margin-top: -3.2%;
+        padding: 0 6.4%;
+        font-family: 'Poppins', Arial, Helvetica, sans-serif;
+        font-style: normal;
+        font-weight: 400;
+        font-size: 16px;
+        line-height: 24px;
+        color: #fff;
+        display: flex;
+        flex: none;
+        order: 2;
+        flex-grow: 0;
+      }
+
+      .links {
+        width: 42.7%;
+        margin: 0 6.4%;
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        justify-content: space-around;
+        padding: 0;
+        flex: none;
+        order: 3;
+        flex-grow: 0;
+
+        .socials {
+          list-style: none;
+        }
+      }
+    }
+  }
+
+  .section-2 {
+    display: flex;
+    margin: 0 auto;
+    width: 100%;
+    background-color: #fff;
+    margin-top: 100px;
+
+    .recent-article {
+      width: 87.2%;
+      margin: 0 auto;
+      display: flex;
+      flex-direction: column;
+    }
+
+    .recent-text {
+      font-family: 'Crete Round', Arial, Helvetica, sans-serif;
+      font-style: normal;
+      font-weight: 400;
+      font-size: 40px;
+      color: #1c1a19;
+    }
+
+    .recent-break {
+      border-bottom: 0.5px solid black;
+      width: 87.2%;
+      margin: 0 auto 18.7% auto;
+    }
+
+    .recent-text-2 {
+      font-family: 'Crete Round', Arial, Helvetica, sans-serif;
+      font-style: normal;
+      font-weight: 400;
+      font-size: 32px;
+      margin-bottom: -5px;
+      color: #091e42;
+    }
+
+    .recent-text-3 {
+      color: #091e42;
+      font-family: 'Inter', Arial, Helvetica, sans-serif;
+      font-style: normal;
+      font-weight: 400;
+    }
+
+    .recent-btns {
+      display: flex;
+      justify-content: space-between;
+      flex-direction: row;
+      list-style: none;
+      margin-left: -10.7%;
+
+      .upper-list {
+        .recent-css-btn,
+        .recent-html-btn,
+        .recent-bootstrap-btn,
+        .recent-ruby-btn {
+          height: 40px;
+          background: #fff;
+          color: #091e42;
+          border: 1px solid #8993a4;
+          margin-bottom: 12px;
+        }
+
+        .recent-css-btn {
+          width: 57px;
+
+          &:hover {
+            background-color: #fff5e1;
+            border: 1px solid #f55800;
+            color: #f55800;
+          }
+
+          &:disabled {
+            background: #f4f1f0;
+            color: #5b5957;
+          }
+        }
+
+        .recent-html-btn {
+          width: 69px;
+
+          &:hover {
+            background-color: #fff5e1;
+            border: 1px solid #f55800;
+            color: #f55800;
+          }
+
+          &:disabled {
+            background: #f4f1f0;
+            color: #5b5957;
+          }
+        }
+
+        .recent-bootstrap-btn {
+          width: 95px;
+
+          &:hover {
+            background-color: #fff5e1;
+            border: 1px solid #f55800;
+            color: #f55800;
+          }
```

Comment:
- [ ] Great job adding the hover effect to your buttons. As suggested by the previous reviewer, kindly add the pressed state also. You can do this using the `:active` pseudoclass.
![image](https://github.com/2004-okari/MY-PORTFOLIO/assets/94551484/091c5b2a-c68d-4e88-a0b3-d6132626a0fd)